.github/scripts/release_notes/index.js

```diff
@@ -25,11 +25,11 @@ if (GITHUB_TOKEN === undefined) {
 
 const DRY_RUN = process.argv.includes("--dry-run")
 
-// Before running the script, make sure the default time zone is set to PST in the GitHub action
-// TODO: For some reason, the code below to update the timezone of the Github Action runners is not working. Ignore this step for now and will fix it later.
-// if (!setTimeZoneToPST()) {
-//     throw new Error("The default time zone is not set to PST")
-// }
+
+const offset = new Date().getTimezoneOffset()
```

Comment:
Can you remove these logs?

Reply:
These logs were utilized for manual testing and validation. For some reason, setting the timezone value in `process.env.TZ` does not function correctly for CI runners. Just in case the `szenius/set-timezone@v2.0` action fails to update the timezone quietly, it's intended to keep these logs for future debugging.